### PR TITLE
upgrade to latest stable rabbitmq

### PR DESCRIPTION
**Pull request by supersven**
Use latest stable RabbitMQ version (`3.13.7`) and Helm chart (`14.6.9`) - one is implied by the other.

This is compatible to our backend:
- I ran tests locally against a docker-compose deployment
- The Helmfile deployment for the CI test runs uses the chart: https://github.com/wireapp/wire-server/blob/sventennie%2Fupgrade-to-latest-stable-rabbitmq/hack/helmfile.yaml#L160

## Checklist

 - [x] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

**Review by lwille — APPROVED** [review]
lgtm

side note: as this is a major version upgrade, I think we should to communicate this as well as [upgrade instructions](https://github.com/bitnami/charts/tree/main/bitnami/rabbitmq#to-1400) to the users of our chart.

**Comment by supersven** [comment]
> ⚠️ side note: as this is a major version upgrade, I think we should to communicate this as well as [upgrade instructions](https://github.com/bitnami/charts/tree/main/bitnami/rabbitmq#to-1400) to the users of our chart.
> 
> lgtm

Thanks a lot :+1: That's a good one. I'll add that to the release notes.

**Comment by lwille** [comment]
> Thanks a lot 👍 That's a good one. I'll add that to the release notes.

Right, the major version upgrade is just in the Helm chart, not in RabbitMQ -- however, there must be reasons for a new major xD

File: changelog.d/2-features/upgrade-rabbitmq [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Use latest stable RabbitMQ version (`3.13.7`) and Helm chart (`14.6.9`). Please
+note that this major version upgrade (`3.11.x` to `3.13.x`) may need special
```
Comment (lwille) [inline changelog.d/2-features/upgrade-rabbitmq:2]: this is a bit misleading: 3.11.x to 3.13.x is a _minor_ version upgrade.

```suggestion
note that this minor version upgrade (`3.11.x` to `3.13.x`) may need special
```

**Pull request merged**